ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -180,13 +193,30 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("start") {
-			pending("should immediately begin sending events") {
+			it("should immediately begin sending events") {
+				var isSended = false
+				let signalProducer = SignalProducer<Int, NoError>(value: 0)
+				signalProducer.start(next:{ _ -> Void in isSended = true })
+				
+				expect(isSended).to(beTruthy())
 			}
 
-			pending("should send interrupted if disposed") {
+			it("should send interrupted if disposed") {
+				var interrupted = false
+				let (producer, sink) = SignalProducer<Int, NoError>.buffer()
+				let disposable = producer.start(interrupted:{ interrupted = true })
+				
+				expect(interrupted).to(beFalsy())
+				disposable.dispose()
+				expect(interrupted).to(beTruthy())
 			}
 
 			pending("should release sink when disposed") {
+				let (producer, sink) = SignalProducer<Int, NoError>.buffer()
+				let disposable = producer.start()
+				
+				disposable.dispose()
+				expect(sink).to(beNil())
```

Comment:
This won't work directly, because this code has a strong reference to `sink`. Instead, you'll need to store the sink in a [weak variable](https://github.com/335g/ReactiveCocoa/blob/786a696f911ae16c3ebfb040a21fc78b5eb7a364/ReactiveCocoaTests/Swift/SignalSpec.swift#L209-L228).